### ISSUE TITLE
Training/Transferring on CIFAR10

### ISSUE THREAD

**Issue by nimaous** [facebookresearch/dino#81]
Hi

Thanks for your nice work. I wonder if you can share the hyperparameter for transfer learning on CIFAR10. Have you succeeded to train on cifar10 from scratch without transferring? if so would you also kindly share the hyperparameters for that?

**Comment by mathildecaron31** [comment]
Hi @nimaous thanks for your interest in our work.
Let me cc @TouvronHugo who conducted experiments on CIFAR10.

**Comment by TouvronHugo** [comment]
Hi @nimaous,
The hparams for CIFAR-10 are:

- global batch size = 768
- lr =  7.5e-6 
- epochs = 1000 
- weight decay = 0.05 
- scheduler = cosine  
- image size = 224 x 224
- repeated augmentation
- labels smoothing
- warmup epochs: 5 
- warmup-lr: 1e-6
- RandAugment
- Mixup = 0.8  
- Cutmix =1.0 
- optimizer: Adamw
- Stoch. Depth: 0.1

Don't hesitate if you have any other questions,
Best,

Hugo

**Comment by mathildecaron31** [comment]
FYI here's the checkpoint finetuned on CIFAR-10:  https://dl.fbaipublicfiles.com/dino/cifar100_ViT_B_dino.pth

For training on CIFAR-10 without DINO pretraining please take a look at https://github.com/facebookresearch/deit

**Comment by stanislavfort** [comment]
How are the CIFAR-10 / CIFAR-100 images normalized? Starting from the usual uint8 0 - 255 range over 3 channels, how exactly do you normalized them before feeding them to the model?

**Comment by TouvronHugo** [comment]
In the same way as ImageNet

**Comment by shallowtoil** [comment]
Hi @mathildecaron31  @TouvronHugo, I'm also wondering whether the hyperparameters (lr, epochs number, etc.) are the same for finetuning DINO with ViT-S and DINO with ViT-B? Thanks for your information.

**Comment by srikar2097** [comment]
> Hi @nimaous, The hparams for CIFAR-10 are:
> 
> * global batch size = 768
> * lr =  7.5e-6
> * epochs = 1000
> * weight decay = 0.05
> * scheduler = cosine
> * image size = 224 x 224
> * repeated augmentation
> * labels smoothing
> * warmup epochs: 5
> * warmup-lr: 1e-6
> * RandAugment
> * Mixup = 0.8
> * Cutmix =1.0
> * optimizer: Adamw
> * Stoch. Depth: 0.1
> 
> Don't hesitate if you have any other questions, Best,
> 
> Hugo

Hi @mathildecaron31 and @TouvronHugo ,
Just to clarify, is these hyper-parameters same for all datasets (CIFAR10, CIFAR100, INat18, INat19, Flowers, Cars, ImageNet) mentioned in Table 6 in main paper? Some of these datasets are not small and fine-tuning for 1000 epochs does take a long time.

**Comment by gihanjayatilaka** [comment]
> FYI here's the checkpoint finetuned on CIFAR-10: https://dl.fbaipublicfiles.com/dino/cifar100_ViT_B_dino.pth
> 
> For training on CIFAR-10 without DINO pretraining please take a look at https://github.com/facebookresearch/deit

Hi! Are these SSL trained on Cifar datasets or supervised trained on Cifar datasets?
Also, the URL says Cifar100. 

Is there a webpage with documentation for these pth files?

Thanks in advance!
